camphr/pipelines/transformers/seq_classification.py

```diff
@@ -73,6 +65,15 @@ class TrfForSequenceClassification(
 
     model_cls = TrfSequenceClassifier
 
+    def set_annotations(self, docs: Iterable[Doc], logits: torch.Tensor):
+        probs = torch.softmax(logits, 1)
```

Comment:
For multi-label classification, softmax should not be called. 
(Sorry for no suggestion code)